### PR TITLE
Corrects standard article mobile ad dimensions

### PR DESCRIPTION
**Pull request by ashleyjelks**
This PR fixes the ad dimensions in mobile ads on Standard articles

[Links to GROW-1340](https://artsyproduct.atlassian.net/browse/GROW-1340)

Post-fix:
Screen Shot 2019-06-26 at 2 19 02 PM

Pre-fix:
Screen Shot 2019-06-26 at 2 21 16 PM

### CODE REVIEW

File: src/Components/Publishing/Layouts/StandardLayout.tsx [hunk 1]
```diff
@@ -92,7 +92,7 @@ export class StandardLayout extends React.Component<
   renderTopRailDisplayAd(isMobileAd: boolean) {
     const { areHostedAdsEnabled, article } = this.props
     const adDimension = isMobileAd
-      ? AdDimension.Mobile_TopLeaderboard
+      ? AdDimension.Mobile_InContentMR1
```
Comment (anandaroop) [inline src/Components/Publishing/Layouts/StandardLayout.tsx]: What does MR stand for?

Comment (ashleyjelks) [reply src/Components/Publishing/Layouts/StandardLayout.tsx]: It is language specific to Hashtag Labs, that's the string we have to include in the HTML so on their end they know what size ad to render, and on what page. 

Their documentation outlines some of that https://htlbid.com/clients/artsy/index.html, and I'm working on some for Artsy internal use as well.

File: src/Components/Publishing/Typings.ts [hunk 2]
```diff
@@ -17,11 +17,7 @@ export enum ArticleAdType {
 export enum AdUnit {
   Desktop_TopLeaderboard = "Desktop_TopLeaderboard",
   Desktop_RightRail1 = "Desktop_RightRail1",
-  Desktop_InContentLB1 = "Desktop_InContentLB1",
```
Comment (l2succes) [inline src/Components/Publishing/Typings.ts]: @ashleyjelks are these ad units obsolete?

Comment (ashleyjelks) [reply src/Components/Publishing/Typings.ts]: No, they were never used. When I first created this enum I added all of the types in the https://htlbid.com/clients/artsy/index.html document, but they didn't all correspond to the types outlined in the Jira tickets and design specs.

**Comment by artsyit** [comment]
:rocket: PR was released in v16.17.33 :rocket:

**Pull request merged**